flow/project.py

```diff
@@ -4994,6 +5015,14 @@ class MyProject(FlowProject):
             "to show result for. Defaults to the main module. "
             "(requires pprofile)",
         )
+        parser_status.add_argument(
```

Comment:
Why does this implement a new option and not use `self._add_operation_selection_arg_group(parser_status)`?

Reply:
@bdice any other thoughts?